source/common/http/filter/gzip_filter.cc

```diff
@@ -0,0 +1,186 @@
+#include "common/http/filter/gzip_filter.h"
+
+#include <regex>
+
+#include "common/common/macros.h"
+
+namespace Envoy {
+namespace Http {
+
+static const std::regex& acceptEncodingRegex() {
```

Comment:
use anon namespace rather than 'static' (from google style guide).

Reply:
That's what I initially did, but it got caught in code review. So, I thought it wasn't relevant anymore and ended up moving back to static.

Reply:
re-reading https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables it does not forbid 'static', but within the context of this style guide, everyone I know that writes C++ based on this guide interprets anonymous namespace to be the preferred mechanism here, even though they have the same effect.  I think this interpretation might have been due to an earlier version of the styleguide or because the only example given is with an anonymous namespace.